src/core/GameKeyHandler.ttslua

```diff
@@ -57,6 +58,80 @@ function addDoomToAgenda()
   doomCounter.call("addVal", 1)
 end
 
+-- move the hovered object to the nearest empty slot on the playermat
+function takeCardIntoThreatArea(playerColor, hoveredObject)
+  -- only continue if an unlocked card
+  if hoveredObject == nil
+      or hoveredObject.type ~= "Card"
+      or hoveredObject.hasTag("Location")
+      or hoveredObject.locked then
+    broadcastToColor("Hover a non-location and try again.", playerColor, "Yellow")
+    return
+  end
+
+  local matColor = playmatApi.getMatColor(playerColor)
+  local mat = guidReferenceApi.getObjectByOwnerAndType(matColor, "Playermat")
+  -- do not continue if the threat area is already full
+  if playmatApi.getEncounterCardDrawPosition(matColor, false) == playmatApi.getEncounterCardDrawPosition(matColor, true) then 
+    broadcastToColor("Threat area is full.", playerColor,"Yellow")
+    return 
+  end
+  local pos = playmatApi.getEncounterCardDrawPosition(matColor, false)
+  
+  -- initialize list of objects to move
+  local moveTheseObjects = {}
+  
+  for _, obj in ipairs(searchLib.onObject(hoveredObject, "isTileOrToken")) do
+    table.insert(moveTheseObjects, obj)
+  end
+  
+  -- find out if the original card is on the green or red playmats
+  local result
```

Comment:
Superfluous line

Reply:
I might be doing something wrong here, or it was because of something else, but in testing without this line, the scripted saw a card as being on the green playmat even when it was on the center playmat, as long as that card was previously on the green playmat.

Reply:
I'm talking about the double `local result`